modules/ROOT/pages/vpc-connectivity-methods-concept.adoc

```diff
@@ -8,7 +8,19 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybr
 image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-platform-private-cloud-edition", title="Anypoint Platform PCE"]
 image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
 
-You can connect an Anypoint Virtual Private Cloud (Anypoint VPC) to your private network using an IPsec tunnel, VPC peering, and CloudHub Direct Connect.
+You can connect an Anypoint Virtual Private Cloud (Anypoint VPC) to your private network using the following methods:
```

Comment:
I wonder why this prod name is not in our naming doc: do you know?

Reply:
It used to be...

Reply:
It's in the Term Presentation Reference:

> Anypoint Virtual Private Cloud (Anypoint VPC)Never refer to as only the VPC.